src/js/services/auth-service.js

```diff
@@ -0,0 +1,11 @@
+import { store } from 'stores/app-store';
+import { core } from 'services/core';
+
+import { setUser } from 'actions/user-actions';
+
+export function login(props) {
+    return core().appUsers.init(props).catch((e) => {
+        console.log(e); //eslint-disable-line no-console
```

Comment:
I was about to say the ajax error is not really descriptive... but my console.log is not either...

![image](https://cloud.githubusercontent.com/assets/781844/12201669/a961c026-b5f3-11e5-9771-211318fbe1e2.png)